.gitignore

```diff
@@ -80,6 +80,3 @@ docs/_build/
 
 # PyBuilder
 target/
-
-# group_vars config
-deployment/ansible/group_vars/all
```

Comment:
We should also delete `deployment/ansible/group_vars/all.example` as part of this PR.